### DISCUSSION TITLE
Play multiple video files (video1, video2...) on the internet one after another (video1->video2...) as a single video

### DISCUSSION THREAD

**Discussion by linuxfedora2020** [jhomlala/betterplayer#241] [Q&A]
I want to play multiple video files (video1, video2...) on the internet one after another (video1->video2...) as a single video, I tried to use the Playlist function, but it cannot set the nextVideoDelay to 0, to make it go from one to next video without any delay (Like a continuous video).

What would you suggest me to do? Thanks

**Comment by jhomlala** [comment]
You can use events to detect end of video and then setup new data source on finished event.
https://github.com/jhomlala/betterplayer#listen-to-video-events
